server/views/cms/rest.py

```diff
@@ -0,0 +1,67 @@
+import logging
+from flask import jsonify, request
+import requests
+import html
+
+from server import app
+from server.util.request import api_error_handler
+
+logger = logging.getLogger(__name__)
+BASE_URL = app.config['PAYLOAD_API_URL']
+API_KEY = app.config['PAYLOAD_API_KEY']
+
+
+@app.route('/api/cms/fetch-page-content', methods=['GET'])
```

Comment:
Shouldn't this be something like
```suggestion
@app.route('/api/cms/pages', methods=['GET'])
```

?

1. `fetch-page-content` isn't RESTful.
2. I'm assuming `slug` (or `slugs[]` if we allow multiple) is a required argument?